### ISSUE TITLE
Arguments helper fields 

### ISSUE THREAD

**Issue by bobinush** [MichalLytek/type-graphql#420]
Documentation states that you can declare some helper fields for your resolver arguments [here (docs/resolvers.md#arguments)](https://github.com/MichalLytek/type-graphql/blob/master/docs/resolvers.md#arguments) but in reality this does not work. 
First of you get the error `Cannot find name 'skip'. Did you mean the instance member 'this.skip'`. Adding `this.` will resolve the error but wont make `startIndex` etc to work, it's always its initial values.
Sending a request with `skip: 3, take:2` will resolve in `startIndex=0` and `endIndex=10` when I log the values in the resolver.

**Comment by MichalLytek** [comment]
Right, it's a typo. It should use getter to map its fields to new properties:
```typescript
@ArgsType()
class GetRecipesArgs {
  // ...

  get startIndex(): number {
    return this.skip;
  }
  get endIndex(): number {
    return this.skip + this.take;
  }
```

If you can create a PR for that, I would be grateful 😉

**Comment by MichalLytek** [comment]
Resolved by #422 🔒
